README.md

```diff
@@ -49,6 +49,14 @@ QuestDB release and sample datasets:
 - live trading data from a cryptocurrency exchange
 - geolocations of 250k unique ships over time
 
+| Query                                                                         | Runtime                                                                                                                                                                          |
```

Comment:
nit: not sure if `Runtime` is the right name for the second column. Something like `Time to execute` might be better.

Reply:
You're right, let me change that to be precise.